doc/source/getting-started/qibosoq_lab_cal.rst

```diff
@@ -0,0 +1,486 @@
+Qibosoq - Qibolab - Qibocal
+===========================
+
+In these examples, we will see how to perform some basic qubit calibration experiments using the three levels of abstraction provided by the Qibo ecosystem:
+
+- Qibosoq: basic pulse level and direct RFSoC connection
+- Qibolab: higher pulse level and integrated control of the complete setup
+- Qibocal: experiment level
+
+
+Preparation pre-experiments
+"""""""""""""""""""""""""""
+
+For all the control modes, we are suppossing to have a qibosoq server running on board.
+
+Qibosoq
+-------
+
+For a control through Qibosoq, no particular preparation is required.
+
+In any case, for every experiment, we have to start the script with:
+
+.. code-block:: python
+
+  import json
+  import socket
+  import numpy as np
+  import matplotlib.pyplot as plt
+
+  from qibosoq.client import execute
+  from qibosoq.components.base import (
+      Qubit,
+      OperationCode,
+      Config
+  )
+
+  HOST = "192.168.0.200"
+  PORT = 6000
+
+
+Qibolab
+-------
+
+For Qibolab, we first need to setup the platform. This includes writing the ``platform.py`` and the ``platform.yml`` files. For more detailed instructions to write these experiments, please refer to the Qibolab documentation [ADD LINK].
```

Comment:
I would replace `platform.py` with `my_platform.py` or `platform_name.py` (the underscore as you prefer), at least people will not infer that they have to call the file exactly `platform.py`.

Or otherwise just write it explicitly (that the name has to be always the same platform name, but which one is up to the user).